.gitignore

```diff
@@ -14,7 +14,5 @@ TEST-*.xml
 
 tcvenv
 
-**/go.work
```

Comment:
question: do we want it committed or part of a setup process? Typically its not standard to add this, but I wonder if this is a good example for it?